### PR TITLE
chore(react-popover): modify arrow styles to create a proper triangle

### PR DESCRIPTION
**Pull request by bsunderhus**
## Previous Behavior

While investigating Semi-transparent support https://github.com/microsoft/fluentui/issues/29491, the `Popover` arrow style turned out to be a little problematic, as we're creating a square and rotating it:
image

## New Behavior

1. restyles the arrow creation to ensure a triangle is generated instead of a rotated square (https://css-tricks.com/snippets/css/css-triangle)

image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">35</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">36</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/Avatar-mount_base.html">622</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/Avatar-mount.html">627</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/Button-mount_base.html">303</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/Button-mount.html">289</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/Field-mount_base.html">1129</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/Field-mount.html">1105</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/FluentProvider-mount_base.html">700</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/FluentProvider-mount.html">724</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/FluentProviderWithTheme-mount_base.html">84</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/FluentProviderWithTheme-mount.html">85</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">35</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">36</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">75</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">74</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/MakeStyles-mount_base.html">862</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/MakeStyles-mount.html">868</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/Persona-mount_base.html">1767</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/Persona-mount.html">1689</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/SpinButton-mount_base.html">1401</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/SpinButton-mount.html">1357</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/SwatchPicker-mount_base.html">1562</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31108/perf-test-react-components/SwatchPicker-mount.html">1521</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: change/@fluentui-react-popover-0f00224e-dd41-401d-b085-6c1f1700cb91.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-popover-0f00224e-dd41-401d-b085-6c1f1700cb91.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31108&commitId=309693287974f8e6ef810afc6eb70c86216c05c8&env=prod&clientType=FLUENTUI)


  <details><summary><b>InfoLabel</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.sizes -- open.rest.chromium.png|2377| Changed|

</details>


  <details><summary><b>Popover Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|21| Changed|
|Popover Converged.positioning.positioned popovers.chromium.png|11| Changed|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|315| Changed|

</details>


  <details><summary><b>Positioning</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.arrow - RTL.chromium.png|1381| Changed|
|Positioning.arrow.chromium.png|1386| Changed|

</details>


  <details><summary><b>Tooltip Converged</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|232| Changed|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|27| Changed|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|27| Changed|
|Tooltip Converged.inverted withArrow.chromium.png|22| Changed|
|Tooltip Converged.inverted withArrow - Dark Mode.chromium.png|3| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|21| Changed|
|Tooltip Converged.overflow-wrap withArrow.chromium.png|3| Changed|
|Tooltip Converged.withArrow.chromium.png|3| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Comment by bsunderhus** [comment]
Closing this as this solution does not meet our requirements.

**Pull request closed without merging**